packages/grafana-ui/src/components/Table/TableNG/utils.ts

```diff
@@ -423,7 +423,7 @@ export const getCellLinks = (field: Field, rowIdx: number) => {
     }
   }
 
-  return links;
+  return links.filter((link) => link.href || link.onClick != null);
```

Comment:
This is interesting - it feels like it would probably fix some existing bugs with datalinks that have neither an href nor an onClick behavior, so showing them does nothing?

Reply:
For context, this change is the after math of some merging conflicts with @leeoniya's [PR](https://github.com/grafana/grafana/pull/103666), so it is indeed fixing some issues 😁